CakePHP/Sniffs/Formatting/OneClassPerUseSniff.php

```diff
@@ -15,43 +15,48 @@
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 
+namespace CakePHP\Sniffs\Formatting;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
 /**
  * Ensures the use contains only one class.
  *
  */
-class CakePHP_Sniffs_Formatting_OneClassPerUseSniff implements PHP_CodeSniffer_Sniff
+class OneClassPerUseSniff implements Sniff
```

Comment:
I think this class can be removed PSR2's UseDeclarationSniff already has check this [check](https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/PSR2/Sniffs/Namespaces/UseDeclarationSniff.php#L73)